pkg/liquidity-mining/test/StakelessGaugeCheckpointer.test.ts

```diff
@@ -112,6 +112,11 @@ describe('StakelessGaugeCheckpointer', () => {
         (await currentWeekTimestamp()).sub(WEEK)
       );
     });
+
+    it('returns gauge types', async () => {
+      const gaugeTypesInAdder = await gaugeAdder.getGaugeTypes();
```

Comment:
Could check that the list isn't empty (might be overkill); anyway this shows the function is there.